Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM docker.io/rustembedded/cross:armv7-unknown-linux-gnueabihf-0.2.1
+
+# Force rebuild when our versions change...
+RUN LIBNFC_VERSION=2b5ad9ce0be19fbca5abc04b4ee0b59fb612e590 && \
+    LIBFREEFARE_VERSION=e95406c0d1b417ff6db7ff8ee95df1b5981ec7b5 && \
+    LIBGATEKEEPER_VERSION=dc021430a68c66878bfe363fb85ce277e5a62501
+
+# We need to clone up here because installing armhf openssl breaks ca-certificates (seriously...)
+RUN git clone https://github.com/nfc-tools/libnfc && \
+    git clone https://github.com/nfc-tools/libfreefare && \
+    git clone https://github.com/Mstrodl/libgatekeeper
```

Comment:
TODO: need to update this to point to `ComputerScienceHouse/libgatekeeper` after https://github.com/ComputerScienceHouse/libgatekeeper/pull/2 is merged

Reply:
This turned into a submodule so we're good :)